compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/ReferenceFinder.java

```diff
@@ -812,9 +812,8 @@ public void visit(BLangInvocation invocationExpr) {
             find(invocationExpr.expr);
         }
 
-        find(invocationExpr.requiredArgs);
         find(invocationExpr.annAttachments);
-        find(invocationExpr.restArgs);
```

Comment:
Are rest args also available in argExprs?

Reply:
Just checked. Rest args are there.